src/util/embedTagHelpers.js

```diff
@@ -6,9 +6,18 @@
  *
  */
 import isObject from 'lodash/fp/isObject';
-
 import { isEmpty } from '../components/validators';
 
+export const removeEmptyElementDataAttributes = obj => {
+  const newObject = {};
```

Comment:
Kanskje litt misvisende navn siden du strengt talt ikke fjerner noen elementer? Kan du bruke obj.filter i stedet for å  filtrere bort tomme elementer?